website/content/docs/platform/k8s/vso/sources/vault/index.mdx

```diff
@@ -305,6 +307,80 @@ The Vault Secrets Operator can optionally cache Vault client information such as
 
 The [Encrypted client cache](/vault/docs/platform/k8s/vso/sources/vault/client-cache) guide will walk you through the steps to enable and configure client cache encryption.
 
+## Instant updates <EnterpriseAlert inline="true" />
+
+The Vault Secrets Operator can instantly update Kubernetes Secrets when changes are made in Vault, instead of waiting for the next refresh interval.
+
+**Supported secret types:**
+- VaultStaticSecret (kv-v1, kv-v2)
+
+<Note title="Requires Vault Enterprise 1.16.3+">
+
+The instant updates option requires Vault Enterprise 1.16.3+ due to the use of [Vault Event Notifications][vault-events].
+
+</Note>
+
+### Enable instant updates for a VaultStaticSecret
```

Comment:
nit: this is a pretty detailed howto for the main page. I wonder if it would make sense to factor it out to its own page. Perhaps something to worth considering for the future.

Reply:
Tried moving it to [it's own page](https://vault-8x7udm5jp-hashicorp.vercel.app/vault/docs/platform/k8s/vso/sources/vault/instant-updates) in 91234da112c67ec275344f8377137a87b8c71f24